app/views/posts/index.html.erb

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
- [ ] In addition to this post index view page, you need to create the `app/views/posts/show.html.erb` also to connect with the show action in the posts controller.